docs/hacking/curriculum/vagrantfile.rst

```diff
@@ -48,8 +48,11 @@ You can install these plugins using the following commands:
 Starting Vagrant and Selfmedicate
 ---------------------------------
 
-The vagrantfile calls ``selfmedicate.sh start`` during is provision step.  To initialize antidote, run
-``vagrant up``
+To test antidote in a virtual machine under Vagrant's control, run:
```

Comment:
I'm not a fan of how this is worded - though I do agree the previous version was also not ideal. How about this?

```suggestion
To start the Vagrant environment for selfmedicate, run:
```